app/src/main/java/com/google/jetpackcamera/receiver/ImageCaptureReceiver.kt

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.jetpackcamera.receiver
+
+import android.content.BroadcastReceiver
+import android.content.ContentValues
+import android.content.Context
+import android.content.Intent
+import android.os.Build
+import android.provider.MediaStore
+import com.google.jetpackcamera.MainActivity
+
+class ImageCaptureReceiver : BroadcastReceiver() {
+    override fun onReceive(context: Context, intent: Intent) {
+        val activityIntent = Intent(context, MainActivity::class.java)
+        val externalContentValues = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
+            intent.getParcelableExtra(MediaStore.EXTRA_OUTPUT, ContentValues::class.java)
```

Comment:
Based on the documentation for [ACTION_TAKE_PICTURE](https://developer.android.com/reference/android/provider/MediaStore#ACTION_IMAGE_CAPTURE), the standard contract is to take a `Uri` as the `EXTRA_OUTPUT`, not a `ContentValues`.

This means we'll want to use the `OutputStream` version of `ImageCapture.OutputFileOptions` where we open the Uri with [`ContentResolver.openOutputStream(Uri)`](https://developer.android.com/reference/android/content/ContentResolver.html#openOutputStream(android.net.Uri))

Reply:
This actually can be done via ActivityScenario. ActivityScenario.result will WAIT for the activity that was launched from ActivityScenario.launchActivityForResult(), and we can verify the result code then. Check the updated ImageCaptureDeviceTest